speech/google/cloud/speech/result.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Representation of a Speech API Result."""
+
+from google.cloud.speech.transcript import Transcript
+
+
+class SpeechResult(object):
```

Comment:
I am :-1: on us preserving `alternatives`. Your helper class still traverses all the alternatives and makes them `Transcript` objects. Why do you need `results.append({'alternatives': alternatives})`?